lib/ember-test-helpers/isolated-container.js

```diff
@@ -83,5 +84,8 @@ export default function isolatedContainer(fullNames) {
     var normalizedFullName = resolver.normalize(fullName);
     container.register(fullName, resolver.resolve(normalizedFullName));
   }
-  return container;
+  return {
+    container: container,
+    registry: registry
+  };
 }
```

Comment:
I assumed that only ember-mocha and ember-qunit would be effected, but I could be wrong.